### ISSUE TITLE
Questions about model reproduction

### ISSUE THREAD

**Issue by SitongGong** [cilinyan/VISA#7]
I have re-pretrain the VISA model following the instructions of the paper and your default parameter configurations and I use 4 A800 GPUs with 1 batch size per GPU, the gradient accumulation is set as 32. However, the results for the model of 3 epochs are higher than your paper about 1 point and the results for the 10 epochs are higher about 3 points in the ReVOS datasets. I am curious about that and wonder if it is normal. Thank you for your reply!

**Comment by cilinyan** [comment]
Thanks for your message. Our training environment is quite unstable with frequent interruptions and resumptions, which could be affecting the final results. It's possible that these interruptions are contributing to the observed differences in performance.

**Comment by Lexarymade** [comment]
hey guys, I'm also trying to re-pretrain the VISA model, I'm just curious about your total training time, it seems very slow to me on my device, and is there any way to speed it up?

**Comment by SitongGong** [comment]
Hello, I use 4 NVIDIA-A800 GPUs to reproduce VISA with gradient accumulation set as 32, resulting in a total batch size of 128. The training speed is low on my device, too. Training it for five epochs takes me about 3.5 days. Maybe you can use the Deepspeed combined with flash attention to speed it up.

**Comment by tranxuantuyen** [comment]
Hi @SitongGong How many GB in the NVIDIA-A800 GPUs you used ?

**Comment by SitongGong** [comment]
@tranxuantuyen about 40GB-50GB on the A800 GPUs with the abovementioned configuration.
